lib/components/TriviaComponent.dart

```diff
@@ -66,32 +60,32 @@ class _TriviaComponentState extends State<TriviaComponent> {
   Widget build(BuildContext context) {
```

Comment:
I actually crashed when I got to this page, so I tweaked.
Reduced a bit re: padding and sized boxes

Reply:
Do you know what specifically caused you to crash? I see the code stylistically reformats some things but not anything that I'd expect to cause a crash!

Reply:
I think it was a render overflow error. I was running on desktop, and after we caught up I learned this was written with a mobile focus.